internal/manifests/targetallocator/service.go

```diff
@@ -27,7 +27,7 @@ func Service(params manifests.Params) *corev1.Service {
 	name := naming.TAService(params.OtelCol.Name)
 	labels := Labels(params.OtelCol, name)
 
-	selector := Labels(params.OtelCol, name)
+	selector := SelectorLabels(params.OtelCol)
```

Comment:
to confirm, we should expect no changes for the selector labels right?

Reply:
There is one change - we no longer have `app.kubernetes.io/name` in there, same as for the collector. Think that's worth calling out?

Reply:
ah yeah its probably because we add it as a label on the deployment, just not for the selector (see [here](https://github.com/open-telemetry/opentelemetry-operator/blob/879c8469b09bc83b94c1ecc8b6bf724a47fecc3f/internal/manifests/manifestutils/labels.go#L68)) which seems fine to me. Let's call it out in the changelog notes to make it clear